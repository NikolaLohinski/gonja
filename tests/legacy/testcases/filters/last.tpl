{{ "Test"|last }}
{{ complex.comments|last }}
{{ 5|last }}
{{ true|last }}
{{ nothing|last }}
{{ simple.chinese_hello_world|last }}
