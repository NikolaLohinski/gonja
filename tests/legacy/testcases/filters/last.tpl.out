t
<legacy_test.comment Value>



ç•Œ
